README.md

```diff
@@ -0,0 +1,49 @@
+
+ TexTron Snake
+ ==============================
+
+ TexTron Snake is very simple ascii snake game tributed to classic Tron arcade.
+ It was meant to be developed as a collaborative programming exercise
+ in course on Open Source Systems taught for undergraduate CS students.
+
+ INSTALL
+ --------------------------------------------------
+
+ * Quick instructions:
+
+```
+ $ ./configure
```

Comment:
I tried to follow the steps indicated for the installation but without success.
The command `./configure` generates a message saying that the file or directory does not exist.

Reply:
You're more than right. The directions were only for distribution tarballs.
I added instructions for source obtained from VCS.
Thanks a lot.

Reply:
Thanks for fixing it, but it doesn't seem to be working properly yet.

Using the develop branch, I ran the following commands:
```
./autogen.sh
./configure
make
```
However, I didn't get any results from creating a `bin` or executable folder.

Following the approach of specifying the directory where it will be installed, by running the following commands:
```
./configure --prefix=/home/user/path
make
make install
```
This time, two folders were created in the `/home/user/path` directory, the bin and the share, with the game executable running.